test/integration_tests/test_mat_executor.py

```diff
@@ -0,0 +1,112 @@
+# coding=utf-8
+# Copyright 2018-2020 EVA
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import unittest
+import pandas as pd
+
+from eva.catalog.catalog_manager import CatalogManager
+from eva.models.storage.batch import Batch
+from eva.server.command_handler import execute_query_fetch_all
+from test.util import create_sample_video, file_remove, \
+    copy_sample_video_to_prefix, load_inbuilt_udfs
+from test.util import DummyObjectDetector
+
+NUM_FRAMES = 10
+
+
+class MaterializedViewTest(unittest.TestCase):
+    def setUp(self):
+        # reset the catalog manager before running each test
+        CatalogManager().reset()
+        create_sample_video()
+        copy_sample_video_to_prefix()
+        load_query = """LOAD DATA INFILE 'dummy.avi' INTO MyVideo;"""
+        execute_query_fetch_all(load_query)
+        load_inbuilt_udfs()
+
+    def tearDown(self):
+        file_remove('dummy.avi')
+        file_remove('ua_detrac.mp4')
```

Comment:
Use setUpClass and testDownClass for common setup and teardown once.

Reply:
Done and improved a few more integration test cases